apps/prairielearn/src/ee/lib/ai-question-generation-benchmark.ts

```diff
@@ -183,6 +183,8 @@ export async function benchmarkAiQuestionGeneration({
   }
 
   const serverJob = await createServerJob({
+    courseId: null,
+    userId: user.user_id,
```

Comment:
Nit, I prefer to keep `userId` and `authnUserId` adjacent to each other. Ditto elsewhere.